From cc0ce5f2c6d52f799707a612832f3715572eeab9 Mon Sep 17 00:00:00 2001
From: Marek Kasik <mkasik@redhat.com>
Date: Thu, 27 Nov 2014 15:06:47 +0100
Subject: printers: Remove border around model name

The border was not there in 3.10 and it is not
intended to be there in 3.14.

https://bugzilla.gnome.org/show_bug.cgi?id=740811

diff --git a/panels/printers/printers.ui b/panels/printers/printers.ui
index ecb545f..3891c35 100644
--- a/panels/printers/printers.ui
+++ b/panels/printers/printers.ui
@@ -332,6 +332,7 @@
                         <property name="visible">True</property>
                         <property name="can_focus">True</property>
                         <property name="show_tabs">False</property>
+                        <property name="show_border">False</property>
                         <child>
                           <object class="GtkButton" id="printer-model-button">
                             <property name="use_action_appearance">False</property>
-- 
cgit v0.10.1

